sorting/inversions.cpp

```diff
@@ -0,0 +1,139 @@
+/**
+ * @file  inversion_count.cpp
```

Comment:
It's better not to add the filename, otherwise you'll need to add the full directory.
```suggestion
 * @file
```

Reply:
@Panquesito7 can you tell me the process of enabling, I did enable something in the actions section after which I was able to see all the pull requests in this repository.

Reply:
Good work. Now, to trigger the GitHub Actions is enabled, make a small commit.
Next, we shall see a commit made by GitHub Actions updating `DIRECTORY.md`. 🙂